journalpump/senders/google_cloud_logging.py

```diff
@@ -57,6 +57,12 @@ def send_messages(self, *, messages, cursor):
             msg = json.loads(msg_str)
             timestamp = msg.pop("timestamp", None)
             journald_priority = msg.pop("PRIORITY", None)
+
+            try:
+                msg["MESSAGE"] = json.loads(msg["MESSAGE"])
+            except (json.JSONDecodeError, KeyError, TypeError):
```

Comment:
A bit of a personal preference, but maybe rather use `dict.get` for checking if `MESSAGE` exists and then decode that rather than catch the `KeyError`. I think `MESSAGE` should always contain a string so in that sense maybe better to not catch the `TypeError` as that probably is an actual error case.

Reply:
Also, using `contextlib.suppress` would be more preferable than `try: except: pass`.

Reply:
I had an Ask for Forgiveness approach in mind here, but yes checking for a key before looks better.
I also doubted about TypeError check here, going to remove it.